stuffer/s2n_stuffer_text.c

```diff
@@ -43,24 +43,27 @@ int s2n_stuffer_peek_check_for_str(struct s2n_stuffer *s2n_stuffer, const char *
     return 0;
 }
 
-int s2n_stuffer_skip_whitespace(struct s2n_stuffer *s2n_stuffer)
+int s2n_stuffer_skip_whitespace(struct s2n_stuffer *s2n_stuffer, uint32_t *skipped)
 {
-    int skipped = 0;
+    PRECONDITION_POSIX(s2n_stuffer_is_valid(s2n_stuffer));
+    uint32_t skip = 0;
     while (s2n_stuffer->read_cursor < s2n_stuffer->write_cursor) {
         switch (s2n_stuffer->blob.data[s2n_stuffer->read_cursor]) {
         case ' ':              /* We don't use isspace, because it changes under locales */
         case '\t':
         case '\n':
         case '\r':
             s2n_stuffer->read_cursor += 1;
-            skipped += 1;
+            skip += 1;
```

Comment:
If you tracked the old `read_cursor`, you wouldn't need this value. You could just set `*skipped = s2n_stuffer->read_cursor - initial_read_cursor`

Reply:
Similar to `s2n_stuffer_skip_expected_char`, `skip` allows me to accept a NULL `skipped`; otherwise, I'd need to check for nullness before dereference it and use it as a counter. Also, either way I'd still need an extra `uint32_t` variable. Which way do you think is cleaner?

Reply:
Done.